### ISSUE TITLE
Invalid Syntax? usdview tutorials/Sphere.usd

### ISSUE THREAD

**Issue by juhalindfors** [PixarAnimationStudios/USD#1925]
### Description of Issue
Invalid syntax error when testing build with:
>  ./bin/usdview  extras/usd/tutorials/convertingLayerFormats/Sphere.usd

### Steps to Reproduce
1. Execute default build script: > python3 build_scripts/build_usd.py release
2. Build completes: Success! To use USD, please ensure that you have
3. PATH contains .../USD/release/bin
4. PYTHONPATH contains .../USD/release/lib/python/
5. > ./release/bin/usdview extras/usd/tutorials/convertingLayerFormats/Sphere.usd yields:

```
Traceback (most recent call last):
  File "./release/bin/usdview", line 28, in <module>
    import pxr.Usdviewq as Usdviewq
  File ".../USD/release/lib/python/pxr/Usdviewq/__init__.py", line 32, in <module>
    from .qt import QtWidgets, QtCore
  File ".../USD/release/lib/python/pxr/Usdviewq/qt.py", line 42, in <module>
    PySideModule = GetPySideModule()
  File ".../USD/release/lib/python/pxr/Usdviewq/qt.py", line 31, in GetPySideModule
    from . import attributeValueEditorUI
  File ".../USD/release/lib/python/pxr/Usdviewq/attributeValueEditorUI.py", line 1
    /********************************************************************************
    ^
SyntaxError: invalid syntax
```

Any hints to how to troubleshoot? Issues with the build, the environment or something else?

### System Information (OS, Hardware)

Ubuntu 18.04

### Package Versions

> python3 --version
Python 3.6.9

PySide2 (5.15.2.1)
PyOpenGL (3.1.6)

**Comment by sunyab** [comment]
This is likely a build issue. We've seen this come up before (see #1285) but have not been able to track down the source. Could you please post your `release/build/USD/CMakeCache.txt` file as well as the results of running `which pyside2-uic`? Also, how did you install PySide2? Via apt-get, or pip, or some other package manager?

**Comment by juhalindfors** [comment]
Hello @sunyab, thank you for the quick response.

I installed PySide2 with pip (and pip3). However `which pyside2-uic` returns empty on my system.

This prompted me to look for an alternative installation for PySide2 which I found as a Ubuntu snap package here: https://snapcraft.io/install/pyside2/ubuntu

I also switched to Python 2 as suggested by @adlarkin in #1285. The system has both Python 2 and 3 installed.

After installing the snap of PySide2 and switching to Python 2 results in a working binary of `usdview` tool.

However, I'm still unable to get this to work with python3. If PySide2 is not installed via pip3 then the build script with `python3` won't start, and once pip3 install of PySide2 is present, the earlier reported issue with `usdview` remains.

I'm not familiar enough with Python ecosystem to troubleshoot it further.

I've attached the `CMakeCache.txt` below if it's helpful. Notice that I've shortened some of the local paths with `...`

[CMakeCache.txt](https://github.com/PixarAnimationStudios/USD/files/9005793/CMakeCache.txt)

**Comment by sunyab** [comment]
Interesting -- the `CMakeCache.txt` indicates USD is picking up `/usr/bin/uic`, which I believe is part of the `qt5-core` package and is used for generating C++ code for Qt. That would explain why we're seeing C-style comments in the error message. If you run `apt list --installed` I'd expect to find `qt5-core` in that list somewhere.

pip3 ought to install pyside2-uic in `/usr/local/bin`. Can you see if that exists? In a fresh docker image running ubuntu 20.04 I did the following:

```
> apt-get install python3 python3-pip
...

> which python3
/usr/bin/python3

> python3 -m pip install PySide2
...
Successfully installed PySide2-5.15.2.1 shiboken2-5.15.2.1

> which pyside2-uic
/usr/local/bin/pyside2-uic
```

If it does exist, make sure `/usr/local/bin` is in your PATH so that the build can find it.

**Comment by juhalindfors** [comment]
Hello @sunyab, thanks again for a very fast response.

I noticed also `/usr/bin/uic` and thought it looked suspicious. Running `apt list --installed` on this system returns following QT related packages:

```
qt-at-spi/bionic,now 0.4.0-8 amd64 [installed,automatic]
qt5-gtk-platformtheme/bionic-updates,bionic-security,now 5.9.5+dfsg-0ubuntu2.6 amd64 [installed,automatic]
qtchooser/bionic,now 64-ga1b6736-5 amd64 [installed,automatic]
qtcore4-l10n/bionic,bionic,now 4:4.8.7+dfsg-7ubuntu1 all [installed,automatic]
qttranslations5-l10n/bionic,bionic,now 5.9.5-0ubuntu1 all [installed,automatic]
qtwayland5/bionic,now 5.9.5-0ubuntu1 amd64 [installed,automatic]
```

There's no `/usr/local/bin/pyside2-uic` present even though `pip3 list` lists PySide2 (5.15.2.1) installed. There's only `/usr/bin/pyside-uic` which reports version "PySide User Interface Compiler version 0.2.15, running on PySide 1.2.2." which I assume is the older version installed from Python 2 pip.

Exploring the pip3 install further with `pip3 show -f pyside2` reveals that pyside2-uic is installed to a relative path `../../../bin/pyside2-uic`. Searching this system with `find /home -iname pyside2-uic` reveals a binary in `~/.local/bin/pyside2-uic`. This binary reports its version as "uic 5.15.2".

I've added the ~/.local/bin to PATH and the build completes using Python 3 and `usdview` works as expected. I get warnings as shown below but I assume these are separate dependency versioning issues.

I'll now try building some of the additional plugins that were not part of the default build. Hope the above can help anyone else trying to build on Ubuntu with Python 3, or if it helps to adjust the default USD build script to take into account the peculiarities of PySide2 binary install locations.

Thanks for your help @sunyab !


```
Warning: in operator() at line 75 of .../USD/pxr/imaging/hgiGL/hgi.cpp -- HgiGL minimum OpenGL requirements not met. Please ensure that OpenGL is initialized and supports version 4.5.
Unable to import OpenGL.arrays.numpymodule.NumpyHandler: No numpy module present: No module named 'numpy'
```
